src/mesido/workflows/io/write_output.py

```diff
@@ -347,13 +352,25 @@ def _name_to_asset(name):
                             results[f"{asset.name}__variable_operational_cost"][0]
                             + results[f"{asset.name}__fixed_operational_cost"][0]
                         )
+                        asset_lifetime_opex_breakdown[asset.asset_type] += (
+                            results[f"{asset.name}__variable_operational_cost"][0]
+                            + results[f"{asset.name}__fixed_operational_cost"][0]
+                        ) * asset.attributes["technicalLifetime"]
 
                         tot_variable_opex_cost_euro += results[
                             f"{asset.name}__variable_operational_cost"
                         ][0]
                         tot_fixed_opex_cost_euro += results[
                             f"{asset.name}__fixed_operational_cost"
                         ][0]
+                        tot_lifetime_variable_opex_cost_euro += (
+                            results[f"{asset.name}__variable_operational_cost"][0]
+                            * asset.attributes["technicalLifetime"]
```

Comment:
and also for all the others.
```suggestion
                            * parameters[f"{asset.name}.technical_life"]
```

Reply:
Agree. Updated as suggested